### PR TITLE
Add PullRequest.patch property

### PR DESCRIPTION
**Pull request by jpopelka**
Currently implemented only for PagurePullRequest.

It returns bytes in case there's some binary blob in the patch.

I'd like to use this in https://github.com/packit-service/packit-service/pull/627

### CODE REVIEW

File: ogr/services/pagure/pull_request.py [hunk 1]
```diff
@@ -105,6 +107,14 @@ def created(self) -> datetime.datetime:
     def diff_url(self) -> str:
         return f"{self.url}#request_diff"
 
+    @property
+    def patch(self) -> bytes:
+        patch_url = f"{self.url}.patch"
+        response = get(patch_url)
```
Comment (lachmanfrantisek) [inline ogr/services/pagure/pull_request.py:113]: I am only if we can use the session used across the whole codebase. (`self.__call_api`)

Comment (mfocko) [reply ogr/services/pagure/pull_request.py:113]: `self.__call_api` automatically uses API endpoint, also `patch_url` points to raw file not JSON
maybe  `_call_project_api_raw`?

Comment (jpopelka) [reply ogr/services/pagure/pull_request.py]: Updated, can you check again?

**Comment by jpopelka** [comment]
Can you give final thumbs up?

> Can you please create an issue for other implementation after the merge?

https://github.com/packit-service/ogr/issues/416

**Comment by lachmanfrantisek** [comment]
> Can you give final thumbs up?

Sorry @jpopelka, I missed that.

**Pull request merged**